src/DataTransfer/domain/measurement/value/bool.go

```diff
@@ -6,7 +6,23 @@ import (
 
 type Bool bool
 
-func (b Bool) ToDisplayString() string {
+func (b Bool) ToDisplayUnitsString() string {
+	return b.toString()
+}
+
+func (b *Bool) GetPodUnits() string {
+	return ""
+}

```

Comment:
Is it correct that if this type of data does not need this method, implement it in blank? Is it not better to start all by default in blank and depending on the type use only the necessary ones?

Reply:
Necesita implementarlo porque necesita implementar la interface Value. A cada tipo hay que implementar la función por separado, no puedes implementárselo a varios de golpe